src/test/scala/edu/ie3/simona/model/participant/FixedFeedInModelSpec.scala

```diff
@@ -13,21 +13,34 @@ import edu.ie3.simona.test.common.UnitSpec
 import edu.ie3.simona.test.common.input.FixedFeedInputTestData
 import edu.ie3.util.quantities.PowerSystemUnits
 import edu.ie3.util.quantities.PowerSystemUnits.MEGAVOLTAMPERE
-import edu.ie3.util.scala.quantities.{Kilovoltamperes, Megavoltamperes}
+import edu.ie3.util.scala.quantities.{
+  ApparentPower,
+  Kilovoltamperes,
+  Megavoltamperes,
+}
+import squants.energy.Kilowatts
+import squants.{Dimensionless, Each, Power}
+import tech.units.indriya.quantity.Quantities
```

Comment:
There are some imports that aren't used. Pleae remove them (Manually or with `CTRL + ALT + O`

Reply:
Same for the others